x-pack/test/rule_registry/common/types.ts

```diff
@@ -5,7 +5,14 @@
  * 2.0.
  */
 import { GenericFtrProviderContext } from '@kbn/test';
-import { Alert, AlertTypeParams } from '../../../plugins/alerting/common';
+import {
+  Alert,
+  AlertTypeParams,
```

Comment:
@simianhacker We've been working on renaming `Alert` to `Rule` and `AlertType` to `RuleType`. Maybe you would like to use the new terminology here?